src/dotnet-interactive-vscode-insiders/package.json

```diff
@@ -208,31 +208,31 @@
       },
       {
         "command": "dotnet-interactive.createNewInteractive",
-        "title": ".NET Interactive: Create Interactive Window"
+        "title": "Polyglot Notebook: Create Interactive Window"

```

Comment:
Keep the old name here and instead add a new command with a new name.  The old command will simply forward to the new one.

Reply:
yup, will create new commands contributions and register old commands as delegate but with the dialog.

Reply:
Do the command prefixes have to match the extension name? The extension name was "Polyglot Notebooks" with an s